web-client/src/app/app.module.ts

```diff
@@ -27,6 +28,7 @@ import { TranslocoRootModule } from './transloco/transloco-root.module';
     AkitaNgRouterStoreModule,
     NgxPrinterModule.forRoot({ printOpenWindow: false }),
     ErrorHandlerModule,
+    IonIntlTelInputModule,
```

Comment:
Do we need to import this in the app.module and the register.module?

Reply:
No, thanks!